jenkins/linux_c_option_test.sh

```diff
@@ -64,6 +64,7 @@ declare -a arr=(
     "-Duse_prov_client:BOOL=ON -Dhsm_type_x509:BOOL=OFF -Dhsm_type_riot:BOOL=ON"
     "-Duse_prov_client:BOOL=ON -Dhsm_type_sastoken:BOOL=ON"
     "-Duse_prov_client:BOOL=ON -Dstrict_prototypes:BOOL=ON"
+    "-DcompileOptoin_C=-Wunused-variable -Wmaybe-uninitialize"
```

Comment:
typo
```suggestion
    "-DcompileOption_C="-Wunused-variable -Wmaybe-uninitialize""
```

Reply:
I don't actually remember what the escape character is on `.sh` so double check that locally.

Reply:
I think it's `\`